hazelcast-spring/src/test/java/com/hazelcast/spring/DummyQuorumListener.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.hazelcast.spring;
+
+import com.hazelcast.quorum.QuorumEvent;
+import com.hazelcast.quorum.QuorumListener;
+
+public class DummyQuorumListener implements QuorumListener {
```

Comment:
Use mockity:

QuorumListener listener = mock(QuorumListener.class)

Dummies are fine if you need to add special logic for them. But mockito is great for creating empty implementations.